src/main/java/org/littil/api/coordinates/service/Coordinates.java

```diff
@@ -0,0 +1,10 @@
+package org.littil.api.coordinates.service;
+
+import lombok.Data;
+
+@Data
+public class Coordinates {
+    private Double lat;
+    private Double lon;
+    private String display_name;
```

Comment:
Is the display name not camelcase on purpose for mapping? Otherwise please use camelcase.

Reply:
This name is required for mapping. However, at the moment it's not used so it can also be removed.